lib/puppet/util/errors.rb

```diff
@@ -37,11 +37,11 @@ def adderrorcontext(error, other = nil)
   # @return [String] description of file and line
   def error_context
     if file and line
-      " at #{file}:#{line}"
+      _(" at %{file}:%{line}") % { file: file, line: line }
```

Comment:
😞  the way we did this, there's no way this is going to come through in some languages. sigh